sdk/java-sdk-spring/src/main/scala/kalix/spring/impl/KalixSpringApplication.scala

```diff
@@ -357,12 +357,12 @@ case class KalixSpringApplication(applicationContext: ApplicationContext, config
   }
 
   def start(): Future[Done] = {
-    logger.info("Starting Kalix Server...")
+    logger.info("Starting Kalix Application...")
```

Comment:
I'm quite sure that I had changed that already, but probably reverted without noticing.